Add Prague 2021 landing page (#1457)
writethedocs/www: docs/conf/prague/2021/news/welcome.rst

:template: {{year}}/generic.html

.. post:: March 15, 2021
   :tags: {{shortcode}}-{{year}}, website, tickets

Announcing Write the Docs Prague {{year}}
===========================================

Welcome to the official launch of `Write the Docs Prague {{year}} <https://www.writethedocs.org/conf/prague/{{year}}/>`_,
our fifth virtual conference.
We have been tuning our approach to these events since our first virtual event in August 2020,
and are happy to invite you back to our online space.

We're excited to announce the dates for our next conference:
**Write the Docs Prague 2021 will be on October 3-5, 2021**

This is another virtual conference, which means you can attend,
speak or have a sponsor presence from anywhere in the world.

> The conference will take place during day time in CET_.

Reviewer: ```suggestion
The conference will take place during the day time in CET_ time zone.
```

.. _CET: https://time.is/CET

Call for Proposals
------------------

Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!

> We plan to launch our CFP in early XXXX and have it last until sometime in XXXX.

Reviewer: ```suggestion
We plan to launch our CFP in early May and have it last until sometime in June.
```

Reviewer: Does this timeline make sense? @plaindocs ?

Get started coming up with ideas for the conference now.

Tickets
-------

> Tickets will officially go on sale in **XXXXX {{ year }}**.

Reviewer: ```suggestion
Tickets will officially go on sale in **April {{ year }}**.
```

Reviewer: What do we think about the ticket sales timeline? @ericholscher ?
